frameworks/security.json

```diff
@@ -115,6 +115,12 @@
             "patch": {
                 "name": "Authenticated user has sensitive permissions"
             }
+        },
+        {
+            "controlID": "C-0266",
```

Comment:
Remove

Reply:
It's important that this control run in the security fw, it will be part of an attack path